components/component-stylesheets/cards.css

```diff
@@ -0,0 +1,145 @@
+.card{
+    display: flex;
+    flex-direction: column;
+    max-width: 26rem;
+    border-radius: 1rem;
+    overflow: hidden;
+    position: relative;
+    box-shadow: 0 3px 1rem var(--mr-light-background-color);
+}
+.main{
+    margin-top: 2rem;
+}
+
+.card-info{
+    margin: 0 1.5rem 1.8rem;
+}
+
+.card-header{
+    font-size: 1.5rem;
+    font-weight: 500;
+}
+
+.card-author{
+    font-size: 1.2rem;
+    font-weight: 500;
+    color: var(--mr-dark-default);
+    font-weight: 400;
+}
+
+.card-badge{
+    position: absolute;
+    top: 1rem;
+    background-color: var(--mr-primary-color);
+    color: var(--mr-white-text-color);
+    padding: 0.2rem 0.8rem;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.card-about{
+    font-size: 1.2rem;
+    font-weight: 400;
+    color: var(--mr-dark-default);
+    margin-top: 1rem;
+    font-weight: 700;
+}
+
+.buttons{
+    margin-top: 2rem;
+    display: flex;
+    justify-content: space-around;
+}
+
+.card-btn{
+    border: none;
+    font-size: 1.5rem;
+    cursor: pointer;
+    background-color: var(--mr-light-primary);
+    color: var(--mr-white-text-color);
+    border-radius: 5px;
+    padding: 0.5rem 1rem;
+    font-weight: 700;
+}
+
+.card-btn:hover{
+    background-color: var(--mr-dark-primary);
+}
+
+.card-btn-icon{
+    font-size: 1.8rem;
+    background: none;
+    color: var(--mr-dark-text-color);
+    padding: 0.3rem 0.7rem;
+}
+
+.card-btn-icon:hover{
+    background-color: var(--mr-light-secondary);
+    border-radius: 50%;
+}
+
+/* horizontal */
+.card-horizontal{
+    max-width: 35rem;
+    margin: auto 0;
```

Comment:
Just a query, are you centering it vertically or horizontally.

Reply:
centering vertically, also since the wrapper container for cards was taking the height wrt the parent so gave margin auto to push against the parent div.